src/contracts/protocol/libraries/helpers/Errors.sol

```diff
@@ -104,4 +104,6 @@ library Errors {
   string public constant RESERVE_NOT_IN_DEFICIT = '102'; // The reserve is not in deficit
   string public constant MUST_NOT_LEAVE_DUST = '103'; // Below a certain threshold liquidators need to take the full position
   string public constant USER_CANNOT_HAVE_DEBT = '104'; // Thrown when a user tries to interact with a method that requires a position without debt
+  string public constant CALLER_NOT_ATOKEN_TRANSFER_ADMIN = '300'; // The caller of the function is not the RWA force transfer admin
```

Comment:
nice idea of using a separate "namespace". 1000 instead of 300 looks cleaner imo, but nbd

Reply:
[78c0cfe](https://github.com/aave/aave-v3-horizon/pull/1/commits/78c0cfe181273d953b2a026a5bbbb2dfa152068a)